packages/space-opera/src/components/poster_controls/poster_controls.ts

```diff
@@ -74,35 +73,43 @@ export class PosterControlsElement extends ConnectedLitElement {
   }
 
   async onCreatePoster() {
-    if (!this.modelViewer)
+    const modelViewer = getModelViewer()!;
+    if (!modelViewer)
       return;
     const posterUrl =
-        createSafeObjectURL(await this.modelViewer.toBlob({idealAspect: true}));
-    dispatchSetPoster(posterUrl.unsafeUrl);
-    dispatchSaveCameraOrbit();
+        createSafeObjectURL(await modelViewer.toBlob({idealAspect: true}));
+    reduxStore.dispatch(dispatchSetPoster(posterUrl.unsafeUrl));
+    const currentOrbit = getCameraState(modelViewer).orbit;
+    const currentFieldOfViewDeg = getCameraState(modelViewer).fieldOfViewDeg;
+    reduxStore.dispatch(
+        dispatchSaveCameraOrbit(currentOrbit, currentFieldOfViewDeg));
   }
 
   onDisplayPoster() {
-    if (!this.modelViewer)
+    const modelViewer = getModelViewer()!;
+    if (!modelViewer)
       return;
-    const src = this.modelViewer.src;
+    // TODO, fix such that we aren't modifying modelViewer in here. i.e. add
```

Comment:
I think its okay to directly edit the model viewer element here, just wasn't 100% sure

Reply:
It's your state to manage. Is there a reason the model can't be reloaded by Redux?

Reply:
we are trying to remove the model from redux, because of all of the micro changes that go on with the it that don't get passed through the redux state.